src/firefly/java/edu/caltech/ipac/firefly/server/AlertsMonitor.java

```diff
@@ -30,7 +30,7 @@
      * check for alerts
      * @param forceSend  if true, send alerts regardless
      */
-    public static void checkAlerts(boolean forceSend) {
+    public static Alert checkAlerts(boolean forceSend) {
```

Comment:
Let's add a flag to avoid sending alerts when polling.  As is, alerts are sent but not delivered.  Not sure what happens to them.

Reply:
done.